### PR TITLE
fix: Remove deprecated commands from CLI

### LINKED ISSUE #5831: Remove deprecated code in CLI
## Action Item

The CLI contains commands that are marked as deprecated, e.g., `keptn onboard service`. 
This code should be removed.

### PR DESCRIPTION
**Pull request by odubajDT**
Signed-off-by: odubajDT <ondrej.dubaj@dynatrace.com>

## This PR

- removes deprecated commands from CLI
- changed integration tests not to use deprecated commands

### Related Issues

Fixes #5831

### CODE REVIEW

**Comment by odubajDT** [comment]
Integration tests pass locally, but there are some problems with gitea on upstream, which make the pre-test phase fail.

**Review by oleg-nenashev — DISMISSED** [review]
Do we use any labels to track the breaking changes?

**Comment by odubajDT** [comment]
Integration tests: https://github.com/keptn/keptn/actions/runs/1630454626

File: test/assets/podtato-head/initProject.sh [hunk 1]
```diff
@@ -10,35 +10,39 @@ case "$1" in
     echo keptn create project "${PROJECT}" --shipyard=./shipyard.yaml   
     keptn create project "${PROJECT}" --shipyard=./shipyard.yaml
     ;;
-  "onboard-service")
-    echo "Onboarding keptn service helloservice in project ${PROJECT}"
-    keptn onboard service helloservice --project="${PROJECT}" --chart=helm-charts/helloserver
+  "create-service")
```
Comment (bacherfl) [inline test/assets/podtato-head/initProject.sh:13]: do we actually still need this file?

Comment (odubajDT) [reply test/assets/podtato-head/initProject.sh:13]: good point, I will search if it is used anywhere

Comment (odubajDT) [reply test/assets/podtato-head/initProject.sh]: it's not used, so I removed it. Thanks!

**Review by bacherfl — APPROVED** [review]

**Pull request merged**